package.json

```diff
@@ -8,6 +8,9 @@
   ],
   "version": "15.2.3",
   "homepage": "http://espadrine.github.com/localeval/",
+  "browser": {
```

Comment:
It's actually observed (and documented by) webpack and browserify:

* browserify: https://github.com/substack/node-browserify#browser-field
* webpack: http://webpack.github.io/docs/configuration.html#resolve-packagemains

it just overrides require for specific packages.